livesql/marshal.go

```diff
@@ -69,8 +69,8 @@ func FieldToValue(field *thunderpb.Field) (driver.Value, error) {
 	}
 }
 
-// filterToProto takes a sqlgen.Filter, runs Valuer on each filter value, and returns a thunderpb.SQLFilter.
-func filterToProto(schema *sqlgen.Schema, tableName string, filter sqlgen.Filter) (*thunderpb.SQLFilter, error) {
+// FilterToProto takes a sqlgen.Filter, runs Valuer on each filter value, and returns a thunderpb.SQLFilter.
```

Comment:
could you show an example for expected usage on these by the client code?

Reply:
```
deps := reactive.Dependencies(ctx)
for _, dep := range deps {
  if sqldep, ok := dep.(livesql.QueryDependency); ok {
    p, err := livesql.FilterToProto(schema, sqldep.Table, sqldep.Filter)
    // ship p elsewhere.
  }
}
```